### PR TITLE
[WIP]support passive event

### PR DESCRIPTION
**Pull request by yurited**
https://github.com/yahoo/subscribe-ui-event/issues/42
@redonkulus 

TODO:
- [x] polyfill
- [ ] test

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/lib/listen.js [hunk 1]
```diff
@@ -4,15 +4,18 @@
...
  * Cross-browser addEventListener.
  * @method listen
  * @param {Object} target - The target to add event listener.
  * @param {String} eventType - The event type.
  * @param {Function} handler - The event handler.
+ * @param {Boolean} passive - A Bool indicating whether or not to register mainEvent as passive event
  * @return {Object} The object to be able to remove the handler.
  */
-function listen(target, eventType, handler) {
+function listen(target, eventType, handler, listen, passive) {
```
Comment (kaesonho) [inline src/lib/listen.js]: `function listen(target, eventType, handler, <strike>listen,</strike> passive) {` ?

File: src/globalVars.js [hunk 2]
```diff
@@ -6,11 +6,25 @@
 
 var EventEmitter = require('eventemitter3');
 
+// Test via a getter in the options object to see if the passive property is accessed
+var supportsPassive = false;
```
Comment (redonkulus) [inline src/globalVars.js]: is this the standard way to detect passive support?

**Comment by kaesonho** [comment]
@yurited  any update for this?

**Comment by kaesonho** [comment]
instead of only support passive, should we make it more general, to just support adding an `option` object as the third parameter?  so that we will also get `capture` and `once`

**Comment by redonkulus** [comment]
Fixed by https://github.com/yahoo/subscribe-ui-event/pull/44

**Pull request closed without merging**